generators/app/templates/src/config/api.js

```diff
@@ -20,12 +20,12 @@ const api = create({
 });
 
 // eslint-disable-next-line no-unused-vars, prettier/prettier, @typescript-eslint/no-unused-vars
-export const apiSetup = dispatch => {
+export const apiSetup = callback => {
```

Comment:
instead of callback rename it to `errorCallback` or `onError` in case we want other callbacks for other cases.

Reply:
what if there is a strange case for someone that needs a monitor for an ok response or something like that?

Reply:
But this callback is called only on 401. I'd rename it to `unauthorizedCallback`.
If you need multiple monitors maybe it would be better to call addMonitor directly in the app.